css/types/round.json

```diff
@@ -13,7 +13,14 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "108",
```

Comment:
@ramiy Any particular reason you chose not to include the "preview" as well, as I did in the note: https://github.com/mdn/browser-compat-data/pull/18173/files#r1016055531 ?

The thing is, both are true - it was introduced in 108 behind a preference, but in the current preview (nightly from108) you don't need to set that preference.

Reply:
done